backend/resources/migrations/080-add-topic-review-table.up.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TYPE topic_name AS ENUM (
+  'stakeholder', 'organisation', 'technology', 'policy', 'resource', 'event', 'initiative'
+);
+
+-- FIXME: Ideally, we would rename the existing review_status type to
+-- approval_status, and rename the field on all the topics/resources,
+-- but this is going to be quite a big rename/refactor to do now.
+
+CREATE TYPE reviewer_review_status AS ENUM (
+  'PENDING', 'ACCEPTED', 'REJECTED'
+);
+
+CREATE TABLE review (
+  id serial NOT NULL PRIMARY KEY,
+  topic_name topic_name NOT NULL,
+  topic_id integer NOT NULL,
+  assigned_by integer NOT NULL REFERENCES stakeholder(id),
+  assigned timestamptz DEFAULT now(),
+  reviewer integer NOT NULL REFERENCES stakeholder(id),
+  reviewed timestamptz DEFAULT NULL,
```

Comment:
same here ... it seems like we are defining here the `modified` column ... unless we want to track time with other meaning

Reply:
I was trying to capture the time when a review was submitted, and not necessarily when this row was modified last. But, I think we could switch to using a `modified` field for consistency.